System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is just a fake project/build thing for trickery and nothing that ends up in the final bits/package right?

Should we call that out in a comment above as well?

Reply:
There's a note for that in the `ByReference<T>` file, as I thought that would've probably be the first file new devs would've tried to open when loading the project, as opposed to just double clicking the actual `.csproj` file in a text editor? 🤔

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/dc0d695531a9ecf3e4558939c3c6517a7c8f313b/System.Private.CoreLib/ByReference%7BT%7D.cs#L5-L11

I can move this in the `.csproj` file if you feel it's better, let me know! 😊